UPGRADE-2.0.md

```diff
@@ -29,20 +29,20 @@
    ```diff
    sylius_paypal_shop:
    -   resource: "@SyliusPayPalPlugin/Resources/config/shop_routing.yaml"
-   +   resource: "@SyliusPayPalPlugin/config/shop_routing.yaml"
+   +   resource: "@SyliusPayPalPlugin/config/shop_routes.yaml"
```

Comment:
```suggestion
   +   resource: "@SyliusPayPalPlugin/config/routes/shop.yaml"
```

Reply:
We wanted to be consistent with other plugins, but doing it this way, as proposed, looks more like the Symfony way 🤔